src/Components/Nav.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+const _Nav = ({ users, products, cart, guestCart, auth, orders }) => {
+  const order = orders.filter(order => order.userId === auth.id);
```

Comment:
The fact that we have to go over all possible orders in multiple components is not good. We might imagine if a site had 2 million users, we could not possibly load down all orders and filter through them on the front end looking for the right one.